middlewares/taskRequests.js

```diff
@@ -20,14 +18,6 @@ async function validateUser(req, res, next) {
       return res.boom.conflict("User does not exist");
     }
 
-    const { userStatusExists, data: userStatus } = await userStatusModel.getUserStatus(userId);
-    if (!userStatusExists) {
```

Comment:
Your PR description says
>  Removes validation where task request for a user was not allowed to be approved when their user status was ACTIVE.

But this if clause stands for a different use case where the user status does not exist.

Reply:
No point in returning an error for this as user status is created when a user without user status is found.